### PR TITLE
registry.get_job_ids should add timestamp

### PR DESCRIPTION
**Pull request by Iliker**
registry.get_job_ids can not custom cleanup time if no timestamp.

### CODE REVIEW

File: rq/registry.py [hunk 1]
```diff
@@ -130,16 +130,16 @@ def __init__(self, name='default', connection=None):
         super(DeferredJobRegistry, self).__init__(name, connection)
         self.key = 'rq:deferred:{0}'.format(name)
 
-    def cleanup(self):
+    def cleanup(self, timestamp=None):
```
Comment (selwin) [inline rq/registry.py]: This method is not used. Could you please revert this change?

File: rq/registry.py [hunk 2]
```diff
@@ -50,9 +50,9 @@ def get_expired_job_ids(self, timestamp=None):
         return [as_text(job_id) for job_id in
                 self.connection.zrangebyscore(self.key, 0, score)]
 
-    def get_job_ids(self, start=0, end=-1):
+    def get_job_ids(self, start=0, end=-1, timestamp=None):
         """Returns list of all job ids."""
-        self.cleanup()
+        self.cleanup(timestamp=timestamp)
```
Comment (selwin) [inline rq/registry.py]: This change should be reverted.

**Comment by theodesp** [comment]
@Iliker Can you explain a little bit more why do we need to add this change?

**Comment by selwin** [comment]
Closing this PR due to inactivity.
